src/more-infos/more-info-cover.html

```diff
@@ -0,0 +1,123 @@
+<link rel='import' href='../../bower_components/polymer/polymer.html'>
+
+<link rel="import" href="../../bower_components/iron-flex-layout/iron-flex-layout-classes.html">
+<link rel="import" href="../../bower_components/paper-icon-button/paper-icon-button.html">
+<link rel='import' href='../../bower_components/paper-slider/paper-slider.html'>
+
+<dom-module id='more-info-cover'>
+  <template>
+    <style is="custom-style" include="iron-flex"></style>
+    <style>
+      .current_position, .current_tilt_position {
+        max-height: 0px;
+        overflow: hidden;
+      }
+      .has-current_position .current_position,
+      .has-current_tilt_position .current_tilt_position {
+        max-height: 90px;
+      }
+    </style>
+
+    <div class$='[[computeClassNames(stateObj)]]'>
+
+      <div class='current_position'>
```

Comment:
Sure, but the rollershutter platform does. It should be hidden like the tilt if not present.